src/lib/collab/stack-client.js

```diff
@@ -383,5 +387,28 @@ export class ServiceClient {
       telepointerDoc
     )
   }
+
+  async postImage(name, noteId, file) {
+    return this.stackClient.fetchJSON(
+      'POST',
+      this.path(noteId, `images?Name=${name}`),
+      file,
+      { headers: { 'Content-Type': 'image/jpeg' } }
+    )
+  }
+
+  async getImages(noteId) {
+    return this.stackClient.fetchJSON(
+      'GET',
+      `http://cozy.localhost:8080/data/io.cozy.notes.documents/${noteId}/relationships/references`
```

Comment:
This is not specific to this PR but maybe we should extract the doctype `io.cozy.notes.documents` to an app wide constant.

Reply:
Updated